presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -83,14 +86,15 @@
     private QueryRunner queryRunner;
     private H2QueryRunner h2QueryRunner;
     private SqlParser sqlParser;
+    private final Closer afterClassCloser = Closer.create();
     private io.prestosql.sql.query.QueryAssertions queryAssertions;
 
     @BeforeClass
     public void init()
```

Comment:
`beforeClass` or something alike to not confuse z before-method initializations

Reply:
Let me avoid doing refactors in this pull requests.